modules/virt-binding-devices-vfio-driver.adoc

```diff
@@ -0,0 +1,137 @@
+<5> The values specified for `mode` determine the permissions for the file.
+<6> The `/etc/modprobe.d/vfio.conf` was created by the `echo "options vfio-pci ids=10de:1eb8 > /etc/modprobe.d/vfio/conf"` command.
+<7> The `echo "dmZpby1wY2kK" | base64 -d` command and the `vfio-pci` commands create the encoded text for `source` as `dmZpby1wY2kK`. The encoded text translates to `vfio-pci`.
+<8> The file that loads the vfio-pci kernel module on the worker nodes.
+
+. Create a `MachineConfig` object file that identifies the kernel argument for the IOMMU configuration:
++
+[source,terminal]
+----
+$ oc create -f 100-worker-kernel-arg-iommu.yaml
+----
+
+. Verify that the `MachineConfig` object file was added.
++
+[source,terminal]
+----
+$ oc get MachineConfig
+----
++
+.Example output
+[source, terminal]
+----
+NAME                             GENERATEDBYCONTROLLER                      IGNITIONVERSION  AGE
+00-master                        d3da910bfa9f4b599af4ed7f5ac270d55950a3a1   3.2.0            25h
+00-worker                        d3da910bfa9f4b599af4ed7f5ac270d55950a3a1   3.2.0            25h
+01-master-container-runtime      d3da910bfa9f4b599af4ed7f5ac270d55950a3a1   3.2.0            25h
+01-master-kubelet                d3da910bfa9f4b599af4ed7f5ac270d55950a3a1   3.2.0            25h
+01-worker-container-runtime      d3da910bfa9f4b599af4ed7f5ac270d55950a3a1   3.2.0            25h
+01-worker-kubelet                d3da910bfa9f4b599af4ed7f5ac270d55950a3a1   3.2.0            25h
+100-worker-iommu                                                            3.2.0            30s
+100-worker-vfiopci-configuration                                            3.2.0            30s
+
+----
+
+. Use the `lspci` command to obtain the `vendor-ID` and the `device-ID` for the PCI device.
++
+[source, terminal]
+----
+$ lspci -nnv | grep -i nvidia
+----
+.Example output
+[source, terminal]
+02:01.0 3D controller [0302]: NVIDIA Corporation GV100GL [Tesla V100 PCIe 32GB] [10de:1eb8] (rev a1)
+
+. Specify the values for `vendor-ID` and `device-ID` and convert the `/etc/modprobe.d/vfio.conf` file to base64 format:
++
+[source,terminal]
+----
+$ echo "options vfio-pci ids=10de:1eb8 | base64
+----
+These values are converted into the `base64 format` and the output is encoded.
++
+.Example output
+[source,terminal]
+----
+b3B0aW9ucyB2ZmlvLXBjaSBpZHM9MTBkZToxZWI4Cg==
+----
+
+. Add the encoded text to `source` in the `MachineConfig` object file.
+
+. Convert `vfio-pci` into base64 format and encode the text:
++
+.Example output
+[source,terminal]
+----
+$ echo vfio-pci | base64
+----
+These values are converted into the `base64 format` and the output is encoded.
++
+.Example output
+[source,terminal]
+----
+dmZpby1wY2kK
+----
+. Add the encoded text to `source` in the `MachineConfig` object file.
```

Comment:
I think there should be a line break before this step.

Reply:
Added.